python_modules/dagster-webserver/dagster_webserver/cli.py

```diff
@@ -162,6 +162,14 @@ def create_dagster_webserver_cli():
     required=False,
     hidden=True,
 )
+@click.option(
+    "--live-data-poll-rate",
+    help="Rate at which the dagster UI polls for updated asset data (defaults to 2 seconds)",
```

Comment:
note that value is actually in ms

Reply:
maybe I just name the arg `--live-data-poll-rate-ms`

Reply:
I added `(in milliseconds)` though... maybe both